public/data/takerukondo.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Takeru Kondo",
+  "type": "personal",
+  "bio": "I am a dev from Kyoto",
+  "avatar": "https://github.com/TakeruKondo.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/TakeruKondo",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/TakeruKondo",
```

Comment:
You have added a link to a _**Suspended Account**_! Please check if you've entered the correct Twitter username in the link, else, update the link with a new one! 

![image](https://user-images.githubusercontent.com/88786642/178637862-139dcc38-6311-40da-b14f-27b276e08a48.png)

Reply:
@TakeruKondo please update or remove the Twitter link